mmedit/datasets/pipelines/augmentation.py

```diff
@@ -145,6 +148,53 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class RandomRotation:
+    """Rotate the image by a randomly-chosen angle, measured in degree.
+
+    Args:
+        keys (list[str]): The images to be rotated.
+        degrees (tuple[float] | tuple[int] | float | int): If it is a tuple,
+        it represents a range (min, max). If it is a float or int,
+        the range is constructed as (-degrees, degrees).
+    """
+
+    def __init__(self, keys, degrees):
+        if isinstance(degrees, float) or isinstance(degrees, int):
+            if degrees < 0.0:
+                raise ValueError(
+                    'If degrees is a single float number, it must be positive.'
+                )
+            else:
+                degrees = (-degrees, degrees)
+        elif mmcv.is_tuple_of(degrees, float) or mmcv.is_tuple_of(
+                degrees, int):
+            degrees = degrees
+        elif degrees is not None:
+            raise TypeError(f'Degrees must be float | int or tuple of float | '
+                            'int, but got '
+                            f'{type(degrees)}.')
```

Comment:
You can skip line 170 to line 172 by using 

```
elif not mmcv.is_tuple_of(degrees, (int, float)):
    raise TypeError(f'Wrong type of "degress": {type(degrees}.')
```

Actually, I think you should also raise an error if degrees is None?